pkg/ccl/backupccl/testdata/backup-restore/in-progress-import-missing-data

```diff
@@ -0,0 +1,368 @@
+# This test induces the skipped span introduction bug described in
+# https://github.com/cockroachdb/cockroach/issues/88042
+# and ensures that RESTORE now fails with an expected error message.
+# The exact timeline that induces the bug is the following:
+#
+# - t0: begin import jobs and pause it
+# - t1: table or database backup with revision history
```

Comment:
nit: wildcard table right, table alone won't be able to resolve the descriptor in the offline state

Reply:
Also do we have table backups in the test below?

Reply:
nah. Will add.